src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt

```diff
@@ -137,6 +141,61 @@ enum class DiscordStatus(private val displayMessageSupplier: Supplier<String>?)
 
     CUSTOM({
         SkyHanniMod.feature.misc.discordRPC.customText.get() // custom field in the config
+    }),
+
+    AUTO({
+        val slayerResult = SLAYER.displayMessageSupplier!!.get()
+        val milestoneResult = try {
+            CROPMILESTONES.displayMessageSupplier!!.get()
+        }
+        catch (e: Exception){
+            "Unable to get milestone"
+        }
+        if (slayerResult != "Planning to do a slayer quest") slayerResult
+        else if (milestoneResult != "Unable to get milestone" && milestoneResult != "Unknown Item") milestoneResult
+        else {
+            val statusNoAuto = DiscordStatus.values().toMutableList()
+            statusNoAuto.removeAt(10)
```

Comment:
This can easily break when changing the order in the list. We should not change the order since it would break the config, but using `statsNoAuto.remove("Auto")` feels better anyway